draft-trammell-rains-protocol.md

```diff
@@ -1247,12 +1222,12 @@ secured with PKIX {{!RFC5280}} certificates.
 |    3 | ee   | End-Entity Certificate           |
 
 A trust anchor certificate constraint specifies a certificate that MUST appear
-as the trust anchor for the certificate presented by the subject of the
-assertion on a connection attempt. An end-entity certificate constraint
-specifies a certificate that MUST be presented by the subject of the assertion
-on a connection attempt.
+as the trust anchor for the certificate presented by the subject of the Singular
```

Comment:
This is stylistic; it's not important to note the singularity of the assertion here. 

```suggestion
as the trust anchor for the certificate presented by the subject of the 
```